cli/cmd/install_test.go

```diff
@@ -39,6 +39,8 @@ func TestRender(t *testing.T) {
 		CreatedByAnnotation:              "CreatedByAnnotation",
 		ProxyAPIPort:                     123,
 		EnableTLS:                        true,
+		TLSTrustAnchorVolumeName:         "linkerd-trust-anchors",
+		TLSSecretsVolumeName:             "linkerd-secrets",
```

Comment:
recommend setting these to something like `TLSTrustAnchorVolumeName`/`TLSSecretsVolumeName`, to disambiguate that the templating is using the injected values and not the constants.

Reply:
Ok gotcha :+1: